nss-cohort-36/bangazon-api-imagination-station-api
# Lr payment type view

# Description

Payment Types view should be finished with list, retrieve, destroy, and create.

Fixes # (issue)

# Testing Instructions

1. In postman, set your method to post, URL to http://127.0.0.1:8000/paymenttypes, and insert this into the body 

{
	"merchant_name": "Silence is Golden",
	"account_number": "45678909876",
	"expiration_date": "11/05/2020"
	
}

Your headers should be 
Content-Type: application/json
Authorization: Token YOUR TOKEN HERE

Make two posts so that you have data to work with.

2. Do a get on http://127.0.0.1:8000/paymenttypes and make sure all payment types show. 
3. Do a get on http://127.0.0.1:8000/paymenttypes/1 and make sure only the payment type withID 1 shows
4. Do a delete on http://127.0.0.1:8000/paymenttypes/1. Check your db to make sure the deleted date is added in the deleted column.
5. Do one more get on http://127.0.0.1:8000/paymenttypes and make sure the deleted item is gone

# Checklist:
